controllers/hospital.controller.js

```diff
@@ -0,0 +1,76 @@
+const Hospital = require("../models/hospital.model"); 
+
+exports.createHospital = async (req, res) => {
+
+    const hospitalObjToBeStoredInDB = {
+        name : req.body.name,
+        address : req.body.address,
+    }
+
+ try {
+    const hospitalCreated = await Hospital.create(hospitalObjToBeStoredInDB);
+
+    console.log("Hospital Created ", hospitalCreated);
+
+    /**
+     * Return the response
+     */
+    const hospitalCreationResponse  = {
+        name : hospitalCreated.name,
+        address : hospitalCreated.address,
+    }
+    return res.status(201).send(hospitalCreationResponse);
+
+} catch(err){
+    console.error("Error while creating new hospital", err.message);
+    return res.status(500).send({
+        message : "some internal error while inserting new hospital"
+    })
+    }
+}
+
+exports.updateHospital = async (req, res) => {
+    try {
+
+    const hospital = await find({
+        _id: req.params.id
+    });
+
+    hospital.name = req.body.name != undefined ? req.body.name: hospital.name;
+    hospital.address = req.body.name != undefined ? req.hospital.address: hospital.address;
+    
+    if(req.body.doctor_ids) {
+        hospital.req.doctor_ids.push(req.body.doctor_ids);
```

Comment:
Good !.. But please also add the support for removing the doctors ids from the hospital